netcdf_to_gltf_converter/config.py

```diff
@@ -108,11 +110,29 @@ class ShiftType(StrEnum):
 class CrsTransformation(BaseModel):
     """The configuration settings for transforming the coordinates."""
 
-    source_epsg: int
-    """int: EPSG code of the source coordinate system."""
+    source_crs: CRS
+    """CRS: The source coordinate system."""
 
-    target_epsg: int
-    """int: EPSG code of the target coordinate system."""
+    target_crs: CRS
+    """CRS: The target coordinate system."""
+
+    @validator("*", pre=True)
+    def validate_epsg(cls, value) -> CRS:
+        if isinstance(value, int):
+            return create_crs(value)
+
+        if isinstance(value, str):
+            return CrsTransformation._create_crs_from_string(value)
+
+        return value
+
+    @staticmethod
+    def _create_crs_from_string(value: str) -> CRS:
+        if "+" in value:
+            parts = value.split("+", maxsplit=1)
```

Comment:
Correct me if im wrong in the naming sense, but maybe this would be more readable:
`horizontal_coordinate, vertical_coordinate = value.split("+", maxsplit=1)`

Reply:
Done